main.go

```diff
@@ -30,8 +44,17 @@ func main() {
 	}
 
 	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
-		w.WriteHeader(*code)
+
+		for _, h := range headerFlags {
```

Comment:
A nice benefit of defining the type as a map is that here you have `for key, values := range …`, which are more expressive than `h` and `ha` 😉

Reply:
done 😸